lib/stat_tracker.rb

```diff
@@ -0,0 +1,26 @@
+class StatTracker
+  def initialize(stat_collections)
+    @games_collection = stat_collections[:games]
+    @teams_collection = stat_collections[:teams]
+    @game_teams_collection = stat_collections[:game_teams]
+  end
+
+  def self.from_csv(locations)
+    StatTracker.new({
```

Comment:
I'm passing the Class as an argument to the `fill_collection` Class method.  This way we don't need a seperate method for each collection file.

Reply:
Great idea @brian-greeson! Looks solid.